vsm

```diff
@@ -118,7 +147,7 @@ class State(object):
 
 
 def emit(signal, value):
-    print('{} = {}'.format(signal, value))
+    Log.i('{} = {}'.format(signal, value))
```

Comment:
I think this will break the tests. You may need to change the comparison there to account for timestamps (by removing them before comparing?)